monoloco/visuals/printer.py

```diff
@@ -360,14 +420,16 @@ def _set_axes(self, ax, axis):
             ax.set_axis_off()
             ax.set_xlim(0, self.width)
             ax.set_ylim(self.height, 0)
-            self.mpl_im0 = ax.imshow(self.im)
+            if not self.args.activities or 'social_distance' not in self.args.activities:
+                self.mpl_im0 = ax.imshow(self.im)
             ax.get_xaxis().set_visible(False)
             ax.get_yaxis().set_visible(False)
 
         else:
             uv_max = [0., float(self.height)]
             xyz_max = pixel_to_camera(uv_max, self.kk, self.z_max)
             x_max = abs(xyz_max[0])  # shortcut to avoid oval circles in case of different kk
```

Comment:
Can you expand on this? Is it gonna become dependant on the focal length?

Reply:
It's something I am not quite sure about. For the moment I hardcoded a value for the bird view width, because before it was distorted by the focal length. I put it at 6 because it is the corresponding width for the default focal length. Instead of hardcoding it I could maybe set x_max as before, according to the focal length, but if it is under 6 (and therefore is too narrow) it would be defaulted back to 6. What do you think ?
Something like :
`x_max = max(abs(xyz_max[0]), 6)`